pkg/apiserver/registry/system/bundle/dump.go

```diff
@@ -0,0 +1,194 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bundle
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"crypto/sha256"
+	"fmt"
+	"io"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/afero"
+)
+
+func dumpFlows(basedir string) error {
+	brListOutput, err := defaultExecutor.Command("ovs-vsctl", "list-br").CombinedOutput()
+	if err != nil {
+		return fmt.Errorf("error when collecting ovs bridge info: %w", err)
+	}
+	err = defaultFS.Mkdir(filepath.Join(basedir, "flows"), os.ModePerm)
+	if err != nil {
+		return fmt.Errorf("error when creating flows output dir: %w", err)
+	}
+	for _, brName := range strings.Split(strings.TrimSpace(string(brListOutput)), "\n") {
+		dumpFlowOutput, err := defaultExecutor.Command("ovs-ofctl", "dump-flows", brName).CombinedOutput()
```

Comment:
You should use OVSCtlClient.DumpFlows() or dumpAntctlGet("ovsflows").

Reply:
Glad to know this, added.